src/main/scala/org/broadinstitute/dsde/firecloud/model/WorkspaceACL.scala

```diff
@@ -75,4 +75,5 @@ object WorkspaceACLJsonSupport {
   implicit val WorkspaceACLFormat = jsonFormat1(WorkspaceACL)
 
   implicit val WorkspaceACLUpdateFormat = jsonFormat2(WorkspaceACLUpdate)
+  implicit val ListOfWorkspaceACLUpdatesFormat = listFormat[WorkspaceACLUpdate]
```

Comment:
if you import `import spray.json.DefaultJsonProtocol._` in WorkspaceApiServiceSpec instead of this, it will work. this has all of the implicits for handling collections (and more). my understanding is that if the application knows how to unmarshal a `List` and it knows how to unmarshal a `WorkspaceACLUpdate`, then it knows how to unmarshal a `List[WorkspaceACLUpdate]`

the reason I bring this up is that this diverges from the Rawls implementation of `WorkspaceACL` (which is where this entire file was pulled from). eventually this code won't exist in orchestration and will be part of a dependency so it can't be different